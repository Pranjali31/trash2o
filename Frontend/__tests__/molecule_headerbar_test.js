import React from 'react';
import { render, fireEvent, waitFor } from '@testing-library/react-native';
import { NavigationContainer } from '@react-navigation/native';
import HeaderBar from '../src/components/molecules/headerBar/index';

test("HeaderBar Component Loading", () => {
    render(<HeaderBar name="molecule_headerbar" type="text" />)

})

test("HeaderBar component snapshot", () => {
    let component = render(<HeaderBar/>).toJSON();
    expect(component).toMatchSnapshot();
})

describe("index Testing component", () => {
    var component;
    beforeEach(() => {
        component = render(<HeaderBar/>).toJSON();

    })

    test("Testing static text data", () => {
        console.log(JSON.stringify(component))
        let styles = component.children[0].children[0];
        expect(styles).toStrictEqual('HeaderBar');
    })
});