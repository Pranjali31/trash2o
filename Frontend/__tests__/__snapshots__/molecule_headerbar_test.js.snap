// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBar component snapshot 1`] = `
<View>
  <Text>
    HeaderBar
  </Text>
</View>
`;
